client/components/site-verticals-suggestion-search/index.jsx

```diff
@@ -181,8 +184,12 @@ export class SiteVerticalsSuggestionSearch extends Component {
 			  } );
 		return (
 			<>
-				<QueryVerticals searchTerm={ inputValue.trim() } debounceTime={ 300 } />
-				<QueryVerticals searchTerm={ DEFAULT_VERTICAL_KEY } />
+				<QueryVerticals
+					searchTerm={ inputValue.trim() }
+					siteType={ siteType }
+					debounceTime={ 300 }
+				/>
+				<QueryVerticals searchTerm={ DEFAULT_VERTICAL_KEY } siteType={ siteType } />
```

Comment:
Should there be varying default fallbacks for each segment? 

Currently we're requesting `business` as a default for every segment:

```
{
    business: {
       business: [{...}],
       ...
    },
    professional: {
       business: [{...}],
       ...
    },
    blog: {
       business: [{...}],
       ...
    },
}

```

Reply:
That's a good note, there totally should be! Glad you created the issue #31, let's track it separately from this PR